pom.xml

```diff
@@ -234,7 +246,90 @@
                     </to>
                 </configuration>
             </plugin>
+
+
+            <plugin>
+                <groupId>org.openapitools</groupId>
+                <artifactId>openapi-generator-maven-plugin</artifactId>
+                <!-- RELEASE_VERSION -->
+                <version>5.2.1</version>
+                <!-- /RELEASE_VERSION -->
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>${project.basedir}/src/main/resources/api-docs.yml</inputSpec>
+                            <modelPackage>org.springframework.samples.petclinic.dto</modelPackage>
+                            <generatorName>spring</generatorName>
+                            <library>spring-boot</library>
+
+                            <!-- Only enabling model generation -->
+                            <modelNameSuffix>Dto</modelNameSuffix>
+                            <generateApiTests>false</generateApiTests>
+                            <generateModelDocumentation>false</generateModelDocumentation>
+                            <generateModelTests>false</generateModelTests>
+                            <generateApis>false</generateApis>
+                            <generateApiDocumentation>false</generateApiDocumentation>
+                            <generateModels>true</generateModels>
+                            <generateSupportingFiles>false</generateSupportingFiles>
+                            <!-- Activating JAVA8 features -->
+                            <configOptions>
+                                <performBeanValidation>true</performBeanValidation>
+                                <dateLibrary>legacy</dateLibrary>
```

Comment:
We could open a PR to move to the `java.time` API ?

Reply:
No , IMO you can wait

Reply:
Ok. I could wait and let you try.

Reply:
I will try by the end of the week and get back to you to see if it's possible

Reply:
Ok thank you. Apart from this point I believe everything is OK for merging your PR.

Reply:
I checked on my computer.
By enabling this property OpenAPIGenerator makes DTOs using ``LocalDate``. 
If I'm not wrong, the version of SpringBoot used in this project depends on JPA 2.2.
IMO, we should create a dedicated PR to enable JAVA 8 time api capabilities. By this way, we could enable it in both model and DTOs classes.

What do you think about this proposal @arey ?

Reply:
I'm agree. https://github.com/spring-petclinic/spring-petclinic-rest/issues/57